Assets/MRTK/StandardAssets/Shaders/MixedRealityTextMeshProSprite.shader

```diff
@@ -0,0 +1,194 @@
+        fixed _ClipSphereSide;
+        float4x4 _ClipSphereInverseTransform;
+#endif
+
+#if defined(_CLIPPING_BOX)
+        fixed _ClipBoxSide;
+        float4x4 _ClipBoxInverseTransform;
+#endif
+
+        uniform float        _VertexOffsetX;
+        uniform float        _VertexOffsetY;
+
+			struct appdata_t
+			{
+                float4 vertex   : POSITION;
+				float4 color    : COLOR;
+				float2 texcoord : TEXCOORD0;
+                UNITY_VERTEX_INPUT_INSTANCE_ID
+			};
+
+			struct v2f
+			{
+                
+                float4 vertex   : SV_POSITION;
+				fixed4 color    : COLOR;
+				half2 texcoord  : TEXCOORD0;				
+#if defined(_CLIPPING_PRIMITIVE)
+                float3 worldPosition    : TEXCOORD1;
+#else 
+                float4 worldPosition : TEXCOORD1;
+#endif           
+                UNITY_VERTEX_INPUT_INSTANCE_ID
+                UNITY_VERTEX_OUTPUT_STEREO     
+			};
+			
+			fixed4 _Color;
+			fixed4 _TextureSampleAdd;
+			float4 _ClipRect;
+
+			v2f vert(appdata_t IN)
+			{
+				v2f OUT;
+
+				UNITY_SETUP_INSTANCE_ID(IN);
+				UNITY_TRANSFER_INSTANCE_ID(IN, OUT);
+				UNITY_INITIALIZE_VERTEX_OUTPUT_STEREO(OUT);
+
+				OUT.worldPosition = IN.vertex;
+				OUT.vertex = UnityObjectToClipPos(OUT.worldPosition);
+
+				OUT.texcoord = IN.texcoord;
+				
+                float4 vertin = IN.vertex;
+                vertin.x += _VertexOffsetX;
+                vertin.y += _VertexOffsetY;
+                float4 vPosition = UnityObjectToClipPos(vertin);
+
+				#ifdef UNITY_HALF_TEXEL_OFFSET
+				OUT.vertex.xy += (_ScreenParams.zw-1.0)*float2(-1,1);
+				#endif
+				
+				OUT.color = IN.color * _Color;
+#if defined(_CLIPPING_PRIMITIVE)
+                OUT.worldPosition = mul(unity_ObjectToWorld, vertin).xyz;
+#endif
+
+				return OUT;
+			}
+
+			sampler2D _MainTex; 
+
+			fixed4 frag(v2f IN) : SV_Target
+			{
+				half4 color = (tex2D(_MainTex, IN.texcoord) + _TextureSampleAdd) * IN.color;
+				
+			#if UNITY_VERSION < 530
+				if (_UseClipRect)
+					color.a *= UnityGet2DClipping(IN.worldPosition.xy, _ClipRect);
+			#else
+				color.a *= UnityGet2DClipping(IN.worldPosition.xy, _ClipRect);
+			#endif
+				
+            // Primitive clipping.
+#if defined(_CLIPPING_PRIMITIVE)
+                float primitiveDistance = 1.0;
+#if defined(_CLIPPING_PLANE)
+                primitiveDistance = min(primitiveDistance, PointVsPlane(IN.worldPosition, _ClipPlane) * _ClipPlaneSide);
+#endif
+#if defined(_CLIPPING_SPHERE)
+                primitiveDistance = min(primitiveDistance, PointVsSphere(IN.worldPosition, _ClipSphereInverseTransform) * _ClipSphereSide);
+#endif
+#if defined(_CLIPPING_BOX)
+                primitiveDistance = min(primitiveDistance, PointVsBox(IN.worldPosition, _ClipBoxInverseTransform) * _ClipBoxSide);
+#endif    
+                color *= step(0.0, primitiveDistance);
+#endif
+
+				#ifdef UNITY_UI_ALPHACLIP
+				clip (color.a - 0.001);
+				#endif
+
+				return color;
+			}
+		ENDCG
+		}
+	}
+

```

Comment:
Some extra whitespace

Reply:
Thanks for catching the extra whitespace. I will remove it!